### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by vivi0204** [Chang-Hongyang/Cas_CLN-BTCAMS#1]
Hi, looking for your help. We, want to ask you for the dataset  which divide the CMeIE according to SEO, EPO, normal? We want to do a comparative experiment, but I can't download CPIH2020 now. The CMeIE downloaded by CBLUE cannot segment by itself consistent dataset statistics from your paper.  Hope to see your reply soon, thank you very much!

**Comment by Chang-Hongyang** [comment]
Hi, you can reference the CasREL paper as well as we do to deal with the dataset splitting. if a sentence doesn't include overlapping triple, it can be seen as normal.&nbsp;
I'm sorry that I'm not in school now, so can't provide the splitting dataset.



---Original---
From: ***@***.***&gt;
Date: Thu, Aug 18, 2022 13:11 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [Chang-Hongyang/Cas_CLN-BTCAMS] dataset (Issue #1)




 
Hi, looking for your help. We, want to ask you for the dataset  which divide the CMeIE according to SEO, EPO, normal? We want to do a comparative experiment, but I can't download CPIH2020 now. The CMeIE downloaded by CBLUE cannot segment by itself consistent dataset statistics from your paper.  Hope to see your reply soon, thank you very much!

**Comment by vivi0204** [comment]
这是来自QQ邮箱的自动回复邮件。你好，我是张维彦，我已经收到你的邮件了，会尽快给您回复。

**Comment by vivi0204** [comment]
OK. Are you using the CMeIE downloaded by CBLUE? Classify by sentence, are you segmenting the samples by periods? We divide the training set, validation set, and test set according to the period to 21155, 5270 and 6590 respectively. Are there any other filtering rules?





------------------ Original ------------------
From: Chang-Hongyang ***@***.***&gt;
Date: Thu,Aug 18,2022 1:30 PM
To: Chang-Hongyang/Cas_CLN-BTCAMS ***@***.***&gt;
Cc: vivi0204 ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [Chang-Hongyang/Cas_CLN-BTCAMS] dataset (Issue #1)





 Hi, you can reference the CasREL paper as well as we do to deal with the dataset splitting. if a sentence doesn't include overlapping triple, it can be seen as normal.&amp;nbsp; 
 I'm sorry that I'm not in school now, so can't provide the splitting dataset. 
 
 
 
 ---Original--- 
 From: ***@***.***&amp;gt; 
 Date: Thu, Aug 18, 2022 13:11 PM 
 To: ***@***.***&amp;gt;; 
 Cc: ***@***.***&amp;gt;; 
 Subject: [Chang-Hongyang/Cas_CLN-BTCAMS] dataset (Issue #1) 
 
 
 
 
   
 Hi, looking for your help. We, want to ask you for the dataset  which divide the CMeIE according to SEO, EPO, normal? We want to do a comparative experiment, but I can't download CPIH2020 now. The CMeIE downloaded by CBLUE cannot segment by itself consistent dataset statistics from your paper.  Hope to see your reply soon, thank you very much!

**Comment by Chang-Hongyang** [comment]
I'm sorry to reply so late. You may be able to access the data at the following website.
https://www.biendata.xyz/




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Chang-Hongyang/Cas_CLN-BTCAMS"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月18日(星期四) 中午1:48
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Chang-Hongyang/Cas_CLN-BTCAMS] dataset (Issue #1)





 OK. Are you using the CMeIE downloaded by CBLUE? Classify by sentence, are you segmenting the samples by periods? We divide the training set, validation set, and test set according to the period to 21155, 5270 and 6590 respectively. Are there any other filtering rules? 
 
 
 
 
 
 ------------------ Original ------------------ 
 From: Chang-Hongyang ***@***.***&amp;gt; 
 Date: Thu,Aug 18,2022 1:30 PM 
 To: Chang-Hongyang/Cas_CLN-BTCAMS ***@***.***&amp;gt; 
 Cc: vivi0204 ***@***.***&amp;gt;, Author ***@***.***&amp;gt; 
 Subject: Re: [Chang-Hongyang/Cas_CLN-BTCAMS] dataset (Issue #1) 
 
 
 
 
 
  Hi, you can reference the CasREL paper as well as we do to deal with the dataset splitting. if a sentence doesn't include overlapping triple, it can be seen as normal.&amp;amp;nbsp;  
  I'm sorry that I'm not in school now, so can't provide the splitting dataset.  
   
   
   
  ---Original---  
  From: ***@***.***&amp;amp;gt;  
  Date: Thu, Aug 18, 2022 13:11 PM  
  To: ***@***.***&amp;amp;gt;;  
  Cc: ***@***.***&amp;amp;gt;;  
  Subject: [Chang-Hongyang/Cas_CLN-BTCAMS] dataset (Issue #1)  
   
   
   
   
     
  Hi, looking for your help. We, want to ask you for the dataset  which divide the CMeIE according to SEO, EPO, normal? We want to do a comparative experiment, but I can't download CPIH2020 now. The CMeIE downloaded by CBLUE cannot segment by itself consistent dataset statistics from your paper.  Hope to see your reply soon, thank you very much!

**Comment by vivi0204** [comment]
Thank you very much.





------------------ Original ------------------
From: Chang-Hongyang ***@***.***&gt;
Date: Tue,Aug 23,2022 4:39 PM
To: Chang-Hongyang/Cas_CLN-BTCAMS ***@***.***&gt;
Cc: vivi0204 ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [Chang-Hongyang/Cas_CLN-BTCAMS] dataset (Issue #1)





 I'm sorry to reply so late. You may be able to access the data at the following website. 
 https://www.biendata.xyz/ 
 
 
 
 
 ------------------&amp;nbsp;原始邮件&amp;nbsp;------------------ 
 发件人:                                                                                                                        "Chang-Hongyang/Cas_CLN-BTCAMS"                                                                                    ***@***.***&amp;gt;; 
 发送时间:&amp;nbsp;2022年8月18日(星期四) 中午1:48 
 ***@***.***&amp;gt;; 
 ***@***.******@***.***&amp;gt;; 
 主题:&amp;nbsp;Re: [Chang-Hongyang/Cas_CLN-BTCAMS] dataset (Issue #1) 
 
 
 
 
 
  OK. Are you using the CMeIE downloaded by CBLUE? Classify by sentence, are you segmenting the samples by periods? We divide the training set, validation set, and test set according to the period to 21155, 5270 and 6590 respectively. Are there any other filtering rules?  
   
   
   
   
   
  ------------------ Original ------------------  
  From: Chang-Hongyang ***@***.***&amp;amp;gt;  
  Date: Thu,Aug 18,2022 1:30 PM  
  To: Chang-Hongyang/Cas_CLN-BTCAMS ***@***.***&amp;amp;gt;  
  Cc: vivi0204 ***@***.***&amp;amp;gt;, Author ***@***.***&amp;amp;gt;  
  Subject: Re: [Chang-Hongyang/Cas_CLN-BTCAMS] dataset (Issue #1)  
   
   
   
   
   
   Hi, you can reference the CasREL paper as well as we do to deal with the dataset splitting. if a sentence doesn't include overlapping triple, it can be seen as normal.&amp;amp;amp;nbsp;   
   I'm sorry that I'm not in school now, so can't provide the splitting dataset.   
     
     
     
   ---Original---   
   From: ***@***.***&amp;amp;amp;gt;   
   Date: Thu, Aug 18, 2022 13:11 PM   
   To: ***@***.***&amp;amp;amp;gt;;   
   Cc: ***@***.***&amp;amp;amp;gt;;   
   Subject: [Chang-Hongyang/Cas_CLN-BTCAMS] dataset (Issue #1)   
     
     
     
     
       
   Hi, looking for your help. We, want to ask you for the dataset  which divide the CMeIE according to SEO, EPO, normal? We want to do a comparative experiment, but I can't download CPIH2020 now. The CMeIE downloaded by CBLUE cannot segment by itself consistent dataset statistics from your paper.  Hope to see your reply soon, thank you very much!

**Comment by Chang-Hongyang** [comment]
> OK. Are you using the CMeIE downloaded by CBLUE? Classify by sentence, are you segmenting the samples by periods? We divide the training set, validation set, and test set according to the period to 21155, 5270 and 6590 respectively. Are there any other filtering rules?
> […](#)
> ------------------ Original ------------------ From: Chang-Hongyang ***@***.***&gt; Date: Thu,Aug 18,2022 1:30 PM To: Chang-Hongyang/Cas_CLN-BTCAMS ***@***.***&gt; Cc: vivi0204 ***@***.***&gt;, Author ***@***.***&gt; Subject: Re: [Chang-Hongyang/Cas_CLN-BTCAMS] dataset (Issue #1) Hi, you can reference the CasREL paper as well as we do to deal with the dataset splitting. if a sentence doesn't include overlapping triple, it can be seen as normal.&amp;nbsp; I'm sorry that I'm not in school now, so can't provide the splitting dataset.
> ---Original--- From: ***@***.***&amp;gt; Date: Thu, Aug 18, 2022 13:11 PM To: ***@***.***&amp;gt;; Cc: ***@***.***&amp;gt;; Subject: [Chang-Hongyang/Cas_CLN-BTCAMS] dataset (Issue #1) Hi, looking for your help. We, want to ask you for the dataset which divide the CMeIE according to SEO, EPO, normal? We want to do a comparative experiment, but I can't download CPIH2020 now. The CMeIE downloaded by CBLUE cannot segment by itself consistent dataset statistics from your paper. Hope to see your reply soon, thank you very much! — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&amp;gt; — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

you can also download the data from CBLUE, https://tianchi.aliyun.com/dataset/dataDetail?spm=5176.22060218.J_2657303350.1.70e81343mI8Zfi&dataId=95414

**Comment by Chang-Hongyang** [comment]
> 

Can you give me the link you used? I'd like to have a checking for it.

**Comment by Chang-Hongyang** [comment]
> I'm sorry to reply so late. You may be able to access the data at the following website. https://www.biendata.xyz/
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "Chang-Hongyang/Cas_CLN-BTCAMS" ***@***.***&gt;; 发送时间:&nbsp;2022年8月18日(星期四) 中午1:48 ***@***.***&gt;; ***@***.******@***.***&gt;; 主题:&nbsp;Re: [Chang-Hongyang/Cas_CLN-BTCAMS] dataset (Issue #1) OK. Are you using the CMeIE downloaded by CBLUE? Classify by sentence, are you segmenting the samples by periods? We divide the training set, validation set, and test set according to the period to 21155, 5270 and 6590 respectively. Are there any other filtering rules?
> ------------------ Original ------------------ From: Chang-Hongyang ***@***.***&amp;gt; Date: Thu,Aug 18,2022 1:30 PM To: Chang-Hongyang/Cas_CLN-BTCAMS ***@***.***&amp;gt; Cc: vivi0204 ***@***.***&amp;gt;, Author ***@***.***&amp;gt; Subject: Re: [Chang-Hongyang/Cas_CLN-BTCAMS] dataset (Issue #1) Hi, you can reference the CasREL paper as well as we do to deal with the dataset splitting. if a sentence doesn't include overlapping triple, it can be seen as normal.&amp;amp;nbsp; I'm sorry that I'm not in school now, so can't provide the splitting dataset.
> ---Original--- From: ***@***.***&amp;amp;gt; Date: Thu, Aug 18, 2022 13:11 PM To: ***@***.***&amp;amp;gt;; Cc: ***@***.***&amp;amp;gt;; Subject: [Chang-Hongyang/Cas_CLN-BTCAMS] dataset (Issue #1) Hi, looking for your help. We, want to ask you for the dataset which divide the CMeIE according to SEO, EPO, normal? We want to do a comparative experiment, but I can't download CPIH2020 now. The CMeIE downloaded by CBLUE cannot segment by itself consistent dataset statistics from your paper. Hope to see your reply soon, thank you very much! — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&amp;amp;gt; — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***&amp;gt; — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***&gt;

https://www.biendata.xyz/competition/chip_2020_2/data/
